src/controllers/userController.ts

```diff
@@ -0,0 +1,34 @@
+import userRepository from '@/repositories/userRepository';
+import { Context } from 'hono';
+
+export const userController = {
+  getById: async (c: Context) => {
```

Comment:
Please extract a context type from a validator that you are using for this route and use this type here to solve the type issue below